Source/DualityTemplates/Templates/SolutionTemplate/Source/Launchers/DualityGame/Program.cs

```diff
@@ -10,10 +8,14 @@ internal static class Program
 		[STAThread]
 		public static void Main(string[] args)
 		{
-			bool isDebugging = System.Diagnostics.Debugger.IsAttached || args.Contains(DualityApp.CmdArgDebug);
-			bool isRunFromEditor = args.Contains(DualityApp.CmdArgEditor);
-			if (isDebugging || isRunFromEditor) ShowConsole();
-			DualityLauncher.Run(args);
+			var launcherArgs = new LauncherArgs(args);
```

Comment:
Please avoid `var` where possible to match the current style guide.

Reply:
I think the style could be a bit better on this. When the type is obvious its not needed to repeat the type again. Thats just extra noise. In the above code its clear that the type is `LauncherArgs` as we just used the constructor.

The editor config actually even has a option for this. We just need to change `csharp_style_var_when_type_is_apparent = false:suggestion` to `csharp_style_var_when_type_is_apparent = true:suggestion`.